schema.sql

```diff
@@ -4,8 +4,8 @@ CREATE TABLE locationtable (
   id SERIAL PRIMARY KEY,
   search_query VARCHAR(255),
   formatted_query VARCHAR(255),
-  latitude INTEGER NOT NULL,
```

Comment:
I'm curious why you didn't want to include the NOT NULL? I think* you can put the not null after the variable type like this:  NUMERIC(10, 7) NOT NULL
Keep up the good work!